.github/workflows/sync.yaml

```diff
@@ -10,8 +10,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - upstream_repo_owner: jameswilsenach
-            upstream_repo_name: FlowFI
+          - upstream_repo_owner: Mills33
+            upstream_repo_name: Basecall2Assembly
```

Comment:
Can you please add a new dictionary (with all 3 fields) instead of replacing the current entry? Thanks!